doc/conf.py

```diff
@@ -120,6 +121,9 @@
 html_theme = "pymc_sphinx_theme"
 html_theme_options = {
     "use_search_override": False,
+    "logo": {
```

Comment:
I would not add this. By default the logo already links to thw homepage _of the version/language being browsed_. If we hardcode this, the logo will always link to the default version (latest or stable, not sure) independently of the version being browsed